neo/test/iotest/test_nwbio.py

```diff
@@ -0,0 +1,262 @@
+#
+"""
+Tests of neo.io.nwbio
+"""
+
+from __future__ import unicode_literals, print_function, division, absolute_import
+import unittest
+import os
+from datetime import datetime
+
+try:
+    from urllib.request import urlretrieve
+except ImportError:
+    from urllib import urlretrieve
+from neo.test.iotest.common_io_test import BaseTestIO
+from neo.core import AnalogSignal, SpikeTrain, Event, Epoch, IrregularlySampledSignal, Segment, Block, ImageSequence
+from neo.utils import get_local_testing_data_folder
+try:
+    import pynwb
+    from neo.io.nwbio import NWBIO
+    HAVE_PYNWB = True
+except (ImportError, SyntaxError):
+    NWBIO = None
+    HAVE_PYNWB = False
+import quantities as pq
+import numpy as np
+from numpy.testing import assert_array_equal, assert_allclose
+
+
+@unittest.skipUnless(HAVE_PYNWB, "requires pynwb")
+class TestNWBIO(unittest.TestCase):
```

Comment:
All our IO must have at least class test that inherits
`BaseTestIO` because it run standard test for all IOs.

Here we will run only specific test.


We also need to push some nwb data in gin.

Reply:
done